src/udev/udevadm-info.c

```diff
@@ -58,8 +58,35 @@ static const char *arg_export_prefix = NULL;
 static usec_t arg_wait_for_initialization_timeout = 0;
 static PagerFlags arg_pager_flags = 0;
 static sd_json_format_flags_t arg_json_format_flags = SD_JSON_FORMAT_OFF;
+static ActionType arg_action = ACTION_QUERY;
```

Comment:
I'd like to name this arg_action_type, to make not confused with sd_device_action_t.

Reply:
Makes sense.